engine/core/IgeEntity.js

```diff
@@ -640,7 +640,8 @@ var IgeEntity = IgeObject.extend({
 
 		return this._backgroundPattern;
 	},
-	createTexture: function (defaultSprite = 0, defaultData) {
+	addToRenderer: function (defaultSprite = 0, defaultData) {
```

Comment:
I think this is a great change, but normally it would be outside of the scope of this PR

Reply:
createTexture is just confusing name now, so i renamed

Reply:
I'm ok with this renaming here, the line is blurred to say with certainty that it is outside of the scope, let's not overthink this one and just keep it in here ;)

Reply:
i agree with keeping it in here. the work is done :D